db/schema.rb

```diff
@@ -65,6 +73,19 @@
     t.index ["slug"], name: "index_organisations_on_slug", unique: true
   end
 
+  create_table "schema_info", id: false, force: :cascade do |t|
```

Comment:
Are these session related changes needed?

Reply:
no! I'll get rid of these and the ones below, they are trapped in there from my local db, good spot!